### PR TITLE
Isabelle Tweaks

### PR DESCRIPTION
**Pull request by elias-michaias**
newfair
Please refer to the [changelog](https://github.com/HDR-Development/HewDraw-Remix/files/9737212/changelog.txt).

### CODE REVIEW

**Comment by yoonikoo** [comment]
Isabelle now has sauce. Has really cool combo routes with the new Up B changes and the new FAIR. Also, with the new jump squat, she is able to do these combos with better, but tighter execution. If there is one thing, the range on FAIR is a bit way too small.  The graphics of FAIR look really clean. However, the SFX of fair is... a little too cheesy for it being a shovel. It sounds to me more like a whip/slap rather than a shovel.

**Comment by elias-michaias** [comment]
Adjusted fair hitbox size, still working on SFX choice.

**Comment by yoonikoo** [comment]
Tested out the new changes. LGTM. Still waiting for more changes such as fishrod nerfs and new SFX.

**Comment by elias-michaias** [comment]
> Tested out the new changes. LGTM. Still waiting for more changes such as fishrod nerfs and new SFX.

Fixed both of these, and a lot more.

**Comment by Kaiokendo661** [comment]
Good shit for giving her some well needed adjustments while still leaving her viable yall did great 👌

File: fighters/shizue/src/acmd/aerials.rs [hunk 1]
```diff
@@ -117,11 +119,83 @@ unsafe fn shizue_attack_air_lw_game(fighter: &mut L2CAgentBase) {
...
+    if is_excute(fighter) {
+        WorkModule::off_flag(boma, *FIGHTER_STATUS_ATTACK_AIR_FLAG_ENABLE_LANDING);
+    }
+}
+
+#[acmd_script( agent = "shizue", script = "effect_attackairf" , category = ACMD_EFFECT , low_priority)]
+unsafe fn shizue_attack_air_f_effect(fighter: &mut L2CAgentBase) {
+    let lua_state = fighter.lua_state_agent;
+    let boma = fighter.boma();
+    frame(lua_state, 9.0);
```
Comment (WuBoytH) [inline fighters/shizue/src/acmd/aerials.rs]: Please generate the effect a frame or two earlier than the hitbox so the animation matches with the end of the attack arc.

**Review by WuBoytH — DISMISSED** [review]

**Comment by hdr-balance-backroom** [comment]
APPROVED

**Review by MadVillainLordQ — APPROVED** [review]
LGTM

**Pull request merged**